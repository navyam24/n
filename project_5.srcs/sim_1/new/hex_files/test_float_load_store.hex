42f6f0b7
97808093
06102223
3dfbe0b7
76c08093
06102423
06402103
0001005b
06802103
0001005b
0000007b
