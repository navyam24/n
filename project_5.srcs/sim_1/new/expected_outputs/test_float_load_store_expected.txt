out.s 0 01000010111101101110100101111000 123.455994
out.s 1 00111101111110111110011101101100 0.123000
